client/src/fluid/FluidEditorView.ml

```diff
@@ -62,7 +62,8 @@ let viewPlayIcon (p : props) (ti : FluidToken.tokenInfo) : Types.msg Html.html =
       Vdom.noNode
 
 
-let toHtml (p : props) : Types.msg Html.html list =
+let toHtml (p : props) (duplicatedRecordFields : (ID.t * StrSet.t) list) :
```

Comment:
@pbiggar I end up with `duplicatedRecordFields: List<Record, Set<DuplicatedFieldName>>`, would like to use `Map` instead of `List`, but I am not so fluent in `ocaml`, the overhead to convert `exprAst` to `Map` is a bit much.
Also passing `duplicatedRecordFields` as second argument than inside `props` to reduce footprint for this fix.

Reply:
Yeah, understandable. Maps are much more complex than they need to be, though the next version of tablecloth has fixes for this, so I think the best bet is to merge as is, and improve when/if we see a performance problem or when there's an opportunity to fix this later.

Reply:
Cool, good to know :+1: